src/test/java/core/basesyntax/dao/StorageDaoImplTest.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax.dao;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class StorageDaoImplTest {
+    private static final StorageDaoImpl storageDao = new StorageDaoImpl();
+    private static final Integer DEFAULT_AGE = 18;
+    private static final String DEFAULT_PASSWORD = "123456";
+    private static final String DEFAULT_LOGIN = "Ukraine";
+    private User defaultUser;
+    
+    @BeforeEach
+    void setUp() {
+        defaultUser = new User();
+        defaultUser.setAge(DEFAULT_AGE);
+        defaultUser.setPassword(DEFAULT_PASSWORD);
+        defaultUser.setLogin(DEFAULT_LOGIN);
+    }
+
+    @AfterEach
+    void clearAfterTest() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void addUser_OK() {
```

Comment:
Actually it's redundant to test that Java List.add() method works well) 
The same for equals method. 
You can just remove this test class